packages/formspree-react/package.json

```diff
@@ -37,16 +37,12 @@
     "@stripe/stripe-js": "^1.35.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.22.1",
-    "@swc/core": "^1.3.61",
```

Comment:
not used

Reply:
Actually, needed "@swc/core is required for es5 target." we can remove once we update `tsconfig` to target `es2015`.